### ISSUE TITLE
Responsive images implementation

### ISSUE THREAD

**Issue by BogdanDogaru** [bbc/simorgh#501]
**Is your feature request related to a problem? Please describe.**
Currently the images presented on article pages have a fixed size.

Using the investigation in #500 we should implement the chosen solution (pending any team discussions) to achieve responsive images with HiDPI (Retina) display support.

**Describe the solution you'd like**
Details TBD. See investigation in #500

**Testing notes**
TBD

**Comment by katikaa** [comment]
Product have defined the outstanding requirements for custom images - let's talk about them in the next workshop/refinement meeting :)

**Comment by BogdanDogaru** [comment]
Blocked on #500

**Comment by pjlee11** [comment]
Blocked on https://github.com/BBC-News/simorgh/issues/971

**Comment by jimjohnsonrollings** [comment]
Closing this issue - will be replaced with another issue as an outcome of the conversation in #971
